index.html

```diff
@@ -9,55 +9,43 @@
     <section class="header">
-        <nav>
-            <div class="logo"><p>My Logo</p></div>
-            <div class="menu"><img src="img/Union.png"></div>
+    
+        <nav class="MenuBar">
+            <div class="logoPlace"> 
+                <div class="logo">
+                    <p>My Logo</p>
+                </div>
+            </div>
+            <div class="menu">
+                <img src="img/Union.png">
+            </div>
         </nav>
-        <div class="welcome">
-            <p>I’m Rashmi.<br>
-              Glad to see<br> 
-              you!</p>
-        </div>
         <div class="intro">
-            <p>I’m a software developer! I can help you<br> 
-                build a product , feature or website Look<br> 
-                through some of my work and<br> 
-                experience! If you like what you see and<br> 
-                have a project you need coded, don’t<br> 
-                hesitate to contact me.
-            </p>
-        </div>
-        <div>
-            <p>LET’S CONNECT</p>
-        </div>
-        <div class="icons">
-            <div><img src="img/Vector.png"></div>
-            <div><img src="img/Linkedin icon (1).png"></div>
-            <div><img src="img/Vector (1).png"></div>
-            <div><img src="img/Vector (3).png"></div>
-            <div><img src="img/Vector (4).png"></div>
-            
-          <!--  <ul class="icons">
-                <li>
-                 <li>
-                    <a><img src="img/Vector (1).png"></a>
-                </li>
-                <li>
-                    <a><img src="img/Vector (3).png"></a>
-                </li>
-                <li>
-                    <a><img src="img/Vector (4).png"></a>
-                </li>
-            </ul>       <a><img src="img/Vector.png"></a>
-                </li>
-                <li>
-                    <a><img src="img/Linkedin icon (1).png"></a>
-                </li>
-            --->
+            <div class="welcome">
+                <p>I’m Rashmi.<br>
+                    Glad to see<br> you!
+                </p>
+            </div>
+            <div class="info">
+                <p>I’m a software developer! I can help you<br> 
+                    build a product , feature or website Look<br> 
+                    through some of my work and<br> 
+                    experience! If you like what you see and<br> 
+                    have a project you need coded, don’t<br> 
+                    hesitate to contact me.
+                </p>
+            </div>
+            <div class="connect">
+              <p>LET'S CONNECT</p>
+            </div>
+            <div class="icons">
+                <div><img src="img/Vector.png"></div>
+                <div><img src="img/Linkedin icon (1).png"></div>
+                <div><img src="img/Vector (1).png"></div>
+                <div><img src="img/Vector (3).png"></div>
+                <div><img src="img/Vector (4).png"></div>
+            </div>
```

Comment:
- [ ] Please make sure the list of social media icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag, (so it should be something like `<ul><li><a href="#url"><img /></li></ul>`).  This will improve your code quatily and will help you link each icons to a link that will lead to your social media profile 😃 👍